content/concepts/long-polling.textile

```diff
@@ -0,0 +1,149 @@
+---
+title: Long Polling - Concepts and Considerations
+section: concepts
+index: 32
+jump_to:
+ Section:
+   - A brief history - How did long polling come about?#a-brief-history-how-did-long-polling-come-about
+   - How does long polling work?#how-does-long-polling-work
+   - Considerations when using long polling#considerations-when-using-long-polling
+   - Open source long polling solutions#open-source-long-polling-solutions
+   - Related transports#related-transports
+   - References and further reading#references-and-further-reading  
+   - Ably and long polling:#ably-and-long-polling
+---
+
+In this article, we'll provide an insight into the technique known as "long polling", how it came to be, how it is implemented, how you might use it in your own applications, and how we use it at Ably.
+
+h2(#a-brief-history-how-did-long-polling-come-about). A brief history - How did long polling come about?
+
+In the early days of the web, HTTP made sense as a simple request-response protocol because it was designed as a way to distribute structured documents to interested readers, each document generally having links to others that could be browsed at the reader's leisure. A request-response model was the right strategy because a browser needed only make a request for the resource that the user wished to consume, and the job was done. The technology of the time was vastly more limited than that of today, and holding a connection open would not only have been a significant issue for web server load management but would have had little-to-no value anyway, concerning the goals and use cases that the early web was designed to solve. Early browsers such as Mosaic and the first version of Netscape Navigator were implemented with this in mind and had no support for JavaScript or other features that might position the web as anything more than a distributed resource-sharing network.
```

Comment:
Some of my comment from the WS concepts article apply here. Also, I think it's important that we link back & forth between our articles.  In addition, we will later need to think about how we build up SEO juice for these articles by linking from other locations, but let's review that after